src/common/common.h

```diff
@@ -52,6 +52,10 @@ typedef enum {
 	OQS_ERROR = -1,
 	/** Used to indicate successful return from function. */
 	OQS_SUCCESS = 0,
+	/** Used to indicate that algorithm is not enabled */
+	OQS_NOT_ENABLED,
```

Comment:
I don't see anywhere that `OQS_NOT_ENABLED` or `OQS_NOT_SUPPORTED` are actually used.